tests/Domains/WhoisTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Domains;
+
+use PHPUnit\Framework\TestCase;
+use UKFast\SDK\Domains\WhoisClient;
+
+class WhoisTest extends TestCase
+{
+    /**
+     * @test
+     * @dataProvider domainsToSanitise
+     */
+    public function testSanitisesDomains($unsanitised, $sanitised)
+    {
+        $this->assertEquals($sanitised, (new WhoisClient())->sanitiseDomain($unsanitised));
+    }
+
+    public function domainsToSanitise()
+    {
+        return [
```

Comment:
Maybe add some more tests;  

- trailing slace: `http://foo.bar/`  
- whitespace on end: `http://foo.bar `  
- both the above `http://foo.bar/ `

Optional above with https as well

Reply:
Done, thanks :)